dependencies/alfont/CMakeLists.txt

```diff
@@ -31,6 +31,10 @@ set(FREETYPE_SOURCES
     "freetype/src/autofit/autofit.c"
 )
 
+if (UNIX)
+  add_definitions(-DALFONT_LINUX)
```

Comment:
Modern CMake prefers target properties. How about my proposal: https://github.com/stefanhendriks/Dune-II---The-Maker/pull/445/files ?